sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/WriteStatsTracker.scala

```diff
@@ -69,7 +69,7 @@ trait WriteTaskStatsTracker {
    * @note This may only be called once. Further use of the object may lead to undefined behavior.
    * @return An object of subtype of [[WriteTaskStats]], to be sent to the driver.
    */
-  def getFinalStats(): WriteTaskStats
+  def getFinalStats(taskCommitTime: Long): WriteTaskStats
```

Comment:
Could you update the function description by adding `@param taskCommitTime`, please?

Reply:
Done